README.md

```diff
@@ -1469,6 +1519,20 @@ Finally, let's implement the chat functionality. For this workshop, we're going
 
     _Note: we must also return CORS headers here (or anywhere else we send a response)._
 
+    <details>
+    <summary><i>Note: Ollama support</i></summary>
+    Change the model to a model you're serving locally, for example:
+
+    ```diff
+    -     model: 'gpt-3.5-turbo-0613',
+    +     model: 'dolphin-mistral',
+    ```
+
+    Also, Ollama supports CORS by default from localhost.
+    If you were calling your Ollama model from a remote host,
+    you would need to allow those origins specifically as per [these docs](https://github.com/ollama/ollama/blob/main/docs/faq.md#how-can-i-allow-additional-web-origins-to-access-ollama).
```

Comment:
CORS should apply here since we are connecting via the backend, right?

Reply:
I find my formulation confusing now that a week has gone by... And I will remove those Ollama-specific details which I found useful to make my code work, but in hindsight are not specific to this tutorial.

More details:
By this I meant that when I dug into Ollama, I saw they had custom logic to allow origins for `localhost` and `127.0.0.1` and `0.0.0.0`: https://github.com/ollama/ollama/blob/e5596c194481a2a2d7483757ad8f6469a8367f8f/server/routes.go#L903-L907 and https://github.com/ollama/ollama/blob/e5596c194481a2a2d7483757ad8f6469a8367f8f/server/routes.go#L931-L938
So this means that in case you're connecting to `localhost:11434` but e.g. serving models on `0.0.0.0`, requests would not be blocked due to CORS checks, even though you could expect this behaviour because they are different hosts https://stackoverflow.com/questions/5256251/are-127-0-0-1-and-localhost-considered-as-two-different-domains-by-browsers.

Reply:
Thanks @louisguitton - I ended up removing the note on CORS entirely since CORS would only apply if you were to connect directly to Ollama's API from the browser (ie. via `fetch()`). Since this guide performs LLM logic in the backend via edge functions, CORS shouldn't apply and I'd prefer to not to encourage modifying CORS origins unless needed (from a security perspective).

If you think I've missed/overlooked something here, please let me know!